python/cudf/cudf/core/index.py

```diff
@@ -1933,6 +1933,39 @@ def floor(self, field):
 
         return self.__class__._from_data({self.name: out_column})
 
+    def round(self, field):
+        """
+        Perform round operation on the data to the specified freq.
+
+        Parameters
+        ----------
+        field : str
```

Comment:
Is there a reason to name this `field` when it's discussed as a frequency `freq` in the docstring?

Reply:
FWIW [here's the pandas API for comparison](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DatetimeIndex.round.html).